spark/datadog_checks/spark/spark.py

```diff
@@ -124,6 +124,22 @@
     'diskUsed': ('spark.rdd.disk_used', INCREMENT)
 }
 
+SPARK_STREAMING_STATISTICS_METRICS = {
+    'avgInputRate': ('spark.streaming.statistics.avg_input_rate', INCREMENT),
```

Comment:
is increment appropriate for all of these? counter is technically deprecated :-/ maybe we should use something else?

Reply:
Good point but I would prefer to do this in another PR unless I need to support more types.

Reply:
There is no guarantee that we'll be able to change the submission method for existing metrics right away. Can you please make these new ones submit using a standard type now, before merging?

Reply:
The problem is that I am not sure about what submission types these metrics should use. I couldn't find proper documentation. Also, all metrics at this point are submitted as increment. So changing the type for old metrics is not going to be easy indeed and for the new one, I don't know what they should be. If you know then please, review the PR and suggest what submission type should be used and where. Until I know for sure, and like I said above, I will keep the code as simple as possible and do the code refactoring to use newer methods in another PR.

Reply:
I kept `numProcessedRecords`, `numReceivedRecords`, `numRetainedCompletedBatches` and `numTotalCompletedBatches` as `monotonic_count` in the end. Everything else should be good as `gauge`